NitroxClient/GameLogic/ItemContainers.cs

```diff
@@ -47,6 +48,9 @@ public void BroadcastItemAdd(Pickupable pickupable, Transform containerTransform
             
             if (packetSender.Send(new EntityReparented(itemId, ownerId)))
             {
+                var iOxygenSource = pickupable.gameObject.GetComponent<IOxygenSource>();
+                var entityMetadataUpdate = GetEntityMetadataUpdateForIOxygenMetadata(iOxygenSource, itemId);
+                if (entityMetadataUpdate != null) packetSender.Send(entityMetadataUpdate);
```

Comment:
This seems like a very specific check when we have methods to extract metadata in a general fashion. Did you test any weird behaviour when sending every extractable metadata?
@tornac1234 what is your opinion on this?